pages/profile/change_password.js

```diff
@@ -7,42 +7,34 @@ import Content from 'components/Content/Content';
 import ChangePasswordForm from 'components/ChangePasswordForm/ChangePasswordForm';
 import styles from '../styles/password_reset.css';
 
-class ChangePassword extends React.Component {
-  state = {
-    didChange: false,
+const ChangePassword = () => {
+  const [didChange, setDidChange] = React.useState(false);
+  const onSuccess = () => {
```

Comment:
Very nitpicky, but can you add some newlines between blocks?

Reply:
done

Reply:
My guess is the commit will come in later? I don't see any changes.

Reply:
Yep. commit will come soon. Just making changes that you requested.